Dashboards/Images/Logos/ThycoticLogo.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 75 75"><defs><linearGradient id="linear-gradient" x1="57.73" y1="41.36" x2="57.73" y2="48.82" gradientUnits="userSpaceOnUse"><stop offset="0.32" stop-color="#82bd41"/><stop offset="1" stop-color="#658d3d"/></linearGradient></defs><rect x="53.62" y="28.07" width="3.03" height="1.28" style="fill:#081f2d"/><rect x="53.61" y="30.2" width="3.08" height="9.18" style="fill:#081f2d"/><polygon points="51.61 28.07 48.56 28.07 48.56 30.21 47.27 30.21 47.27 31.38 48.56 31.38 48.56 39.38 51.61 39.38 51.61 31.38 52.69 31.38 52.69 30.21 51.61 30.21 51.61 28.07" style="fill:#081f2d"/><path d="M66.11,33.17V31.9a1.41,1.41,0,0,0-.91-1.25,6.42,6.42,0,0,0-3-.55,6.41,6.41,0,0,0-3.13.57A2.49,2.49,0,0,0,58,33v3.75A2.42,2.42,0,0,0,59.12,39a6.39,6.39,0,0,0,3.13.58,6.44,6.44,0,0,0,3-.54,1.44,1.44,0,0,0,.91-1.27V36.49H63.36v1.27a.54.54,0,0,1-.31.45,1.54,1.54,0,0,1-.86.21c-.76,0-1.15-.21-1.15-.64v-5.9c0-.42.39-.63,1.15-.63a1.63,1.63,0,0,1,.86.2.57.57,0,0,1,.31.45v1.27Z" style="fill:#081f2d"/><path d="M24.57,37.82c0,.26-.37.38-1.11.38-.89,0-1.33-.15-1.33-.47V30.19H19v7.63c0,1.1,1,1.66,3,1.66l1.06,0,.87,0a1.76,1.76,0,0,0,.66-.12v2.13c0,.16-.11.31-.34.44a1.81,1.81,0,0,1-.89.19,1.7,1.7,0,0,1-.89-.2c-.23-.14-.35-.29-.35-.45V40.13H19V41.4c0,.49.4.91,1.2,1.27a8,8,0,0,0,3.2.53,6.44,6.44,0,0,0,3.13-.57,2.51,2.51,0,0,0,1.09-2.3V30.19H24.57Z" style="fill:#081f2d"/><polygon points="6.6 28.07 3.56 28.07 3.56 30.21 2.04 30.21 2.04 31.38 3.56 31.38 3.56 39.38 6.6 39.38 6.6 31.38 8.11 31.38 8.11 30.21 6.6 30.21 6.6 28.07" style="fill:#081f2d"/><path d="M16.21,30.24a9.34,9.34,0,0,0-2.2-.18,5.06,5.06,0,0,0-1.81.26V28.07H9.31V39.38h2.95V31.31h1.66c.46,0,.69.2.69.61v7.46h2.78V31.49C17.39,30.87,17,30.45,16.21,30.24Z" style="fill:#081f2d"/><path d="M36.16,30.65a6.5,6.5,0,0,0-3-.55,6.44,6.44,0,0,0-3.13.57A2.51,2.51,0,0,0,29,33v3.75A2.44,2.44,0,0,0,30.07,39a6.42,6.42,0,0,0,3.13.58,6.51,6.51,0,0,0,3-.54,1.45,1.45,0,0,0,.9-1.27V36.49H34.32v1.27a.55.55,0,0,1-.32.45,1.52,1.52,0,0,1-.86.21c-.76,0-1.14-.21-1.14-.64v-5.9q0-.63,1.14-.63a1.61,1.61,0,0,1,.86.2.58.58,0,0,1,.32.45v1.27h2.74V31.9A1.41,1.41,0,0,0,36.16,30.65Z" style="fill:#081f2d"/><polygon points="51.25 41.36 56.67 41.36 64.22 48.82 58.38 48.82 51.25 41.36" style="fill:url(#linear-gradient)"/><path d="M45.53,30.54a6.31,6.31,0,0,0-3-.56,6.35,6.35,0,0,0-3.14.58,2.48,2.48,0,0,0-1.08,2.29V36.6a2.43,2.43,0,0,0,1.08,2.27,6.35,6.35,0,0,0,3.14.58,6.46,6.46,0,0,0,3-.54,1.44,1.44,0,0,0,.9-1.26V31.79A1.4,1.4,0,0,0,45.53,30.54Zm-1.84,6v1.07a.57.57,0,0,1-.31.45,1.63,1.63,0,0,1-.86.2c-.77,0-1.15-.21-1.15-.63v-5.9c0-.43.38-.64,1.15-.64a1.54,1.54,0,0,1,.86.21.54.54,0,0,1,.31.45v4.79Z" style="fill:#081f2d"/><polygon points="73.67 36.5 68.05 36.5 69.52 37.96 58.38 48.82 64.22 48.82 72.18 40.59 73.67 42.06 73.67 36.5" style="fill:#82bd41"/></svg>
```

Comment:
Provide Guids for **_id_** attributes, remove data-name.
Refer guidelines here https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md

Section to be considered:
Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets